### PR TITLE
discuss page contents space fix

### PR DESCRIPTION
**Pull request by darthleslin**
1. Increased little bit of line height for the lines in the paragraph.
2. Discuss page break tags removed and padding increased.
3. space between the inner-text-box increased to 32px to maintain the uniformity between inside and outside of the wrappers.

### CODE REVIEW

**Comment by kenjis** [comment]
Can you show the screenshots before and after?

**Comment by darthleslin** [comment]
**BEFORE**
before

**AFTER**
after

@kenjis

**Review by michalsn — APPROVED** [review]

**Review by Arhell — APPROVED** [review]

**Comment by kenjis** [comment]
Thank you!
But your git commit is not verified.
Can you upload your public key to GitHub?
Screenshot 2024-05-30 11 48 27

**Comment by darthleslin** [comment]
> Thank you! But your git commit is not verified. Can you upload your public key to GitHub? Screenshot 2024-05-30 11 48 27

Done.
Thank You.

**Review by MGatner — APPROVED** [review]
Thanks for this attention to detail! The only part I don't prefer is the bullet spacing feels too large (though I do agree a bit more space is warranted). But if others are in favor it's just an opinion.

File: public/assets/css/ci-theme.css [hunk 1]
```diff
@@ -2124,3 +2124,34 @@ html.dark-theme #discuss-icon-holder {
...
+    display: inline-block;
+}
+
+.cta-btn:not(:first-of-type) {
+    margin-left: 1rem;
+}
+
+ul li {
+    margin-top: 1rem;
+    line-height: 1.5em;
```
Comment (kenjis) [inline public/assets/css/ci-theme.css:2144]: How about this? @darthleslin @MGatner 

```suggestion
    line-height: 1em;
```
Screenshot 2024-06-07 9 53 11

Comment (darthleslin) [reply public/assets/css/ci-theme.css:2144]: Looks great 👍.

Comment (MGatner) [reply public/assets/css/ci-theme.css:2144]: Excellent! Thank you for entertaining my nitpicks 🤗

**Review by kenjis — APPROVED** [review]

**Pull request merged**